packages/forklift-console-plugin/webpack.config.ts

```diff
@@ -7,141 +7,183 @@ import CopyPlugin from 'copy-webpack-plugin';
-    static: ['./dist'],
-    host: 'localhost',
-    port: 9001,
-    headers: {
-      'Access-Control-Allow-Origin': '*',
-      'Access-Control-Allow-Methods': 'GET, POST, PUT, DELETE, PATCH, OPTIONS',
-      'Access-Control-Allow-Headers': 'X-Requested-With, Content-Type, Authorization',
-      'Service-Worker-Allowed': '/', // needed to support MockServiceWorker
-    },
-    devMiddleware: {
-      writeToDisk: true,
+      ],
     },
-    client: {
-      progress: true,
+    devServer: {
+      static: ['./dist'],
+      host: 'localhost',
+      port: 9001,
+      headers: {
+        'Access-Control-Allow-Origin': '*',
+        'Access-Control-Allow-Methods': 'GET, POST, PUT, DELETE, PATCH, OPTIONS',
+        'Access-Control-Allow-Headers': 'X-Requested-With, Content-Type, Authorization',
+        'Service-Worker-Allowed': '/', // needed to support MockServiceWorker
+      },
+      devMiddleware: {
+        writeToDisk: true,
+      },
+      client: {
+        progress: true,
+      },
     },
-  },
-  plugins: [
-    new DynamicConsoleRemotePlugin({
-      pluginMetadata,
-      extensions,
-    }),
-    new CopyPlugin({
-      patterns: [{ from: '../locales', to: '../dist/locales' }],
-    }),
-    new EnvironmentPlugin({
-      // DATA_SOURCE: used for testing when no api servers are available
-      // if set to mock, network api calls will be mocked
-      DATA_SOURCE: 'remote',
-      // can be 'RedHat' or 'Konveyor',
-      // note: downstream build are set to: 'RedHat'
-      BRAND_TYPE: 'Konveyor',
-      // NAMESPACE: used only on mock data
-      NAMESPACE: 'konveyor-forklift',
-      // DEFAULT_NAMESPACE: UI forms and modals will fallback to this namespace
-      //                    if no namespace is given by user.
-      // note: downstream build are set to: 'openshift-mtv'
-      DEFAULT_NAMESPACE: 'konveyor-forklift',
-      // NODE_ENV: used to bake debugging information on development builds.
-      NODE_ENV: production ? 'production' : 'development',
-      // PLUGIN_NAME: should be set to the plugin name hardcoded in the
-      //              installation scripts, defaults to 'forklift-console-plugin'.
-      PLUGIN_NAME: pluginMetadata.name,
-    }),
-  ],
-  devtool: 'source-map',
-  optimization: {
-    chunkIds: production ? 'deterministic' : 'named',
-    minimize: production ? true : false,
-    minimizer: [
-      // Keep class names and function names in sources to aid debug and diagnostics of prod builds
-      new TerserPlugin({
-        terserOptions: {
-          keep_classnames: true,
-          keep_fnames: true,
-        },
+    plugins: [
+      new DynamicConsoleRemotePlugin({
+        pluginMetadata,
+        extensions,
+      }),
+      new BannerPlugin({ banner, raw: true }),
+      new CopyPlugin({
+        patterns: [
+          { from: '../locales', to: '../dist/locales' },
+          {
+            from: '../build-info.json.template',
+            to: '../dist/build-info.json',
```

Comment:
can you follow the `@openshift/dynamic-plugin-sdk` [1] example and call the file `build-metadata.json` like we do now [2] in our rollup plugin ?


[1] https://github.com/openshift/dynamic-plugin-sdk/blob/main/packages/common/rollup/rollup-configs.js#L151
[2] https://github.com/kubev2v/forklift-console-plugin/blob/main/config/rollup.base.js#L122

Reply:
Ok